### PR TITLE
2 packages from gw31415/satysfi-chemfml at 1.0.1

### PR DESCRIPTION
**Pull request by gw31415**
A SATySFi Package for chemical formulas.

This pull-request concerns:

* `satysfi-chemfml.1.0.0`
* `satysfi-chemfml-doc.1.0.0`
* Homepage: https://github.com/gw31415/satysfi-chemfml/
* Source repo: git+ https://github.com/gw31415/satysfi-chemfml.git
* Bug tracker: https://github.com/gw31415/satysfi-chemfml/issues
# Automatic follow-ups
Choose follow-up actions.  Do not write anything after this section.
- ~~Add to snapshot `snapshot-develop`~~ (Inconsistent)
- [ ] Add to snapshot `snapshot-develop--1` :: satysfi-chemfml-doc.1.0.1 satysfi-chemfml.1.0.1
- ~~Add to snapshot `snapshot-stable-0-0-4`~~ (Inconsistent)
- ~~Add to snapshot `snapshot-stable-0-0-5`~~ (Inconsistent)
- ~~Add to snapshot `snapshot-stable-0-0-6`~~ (Inconsistent)
- [ ] Add to snapshot `snapshot-stable-0-0-6--1` :: satysfi-chemfml-doc.1.0.1 satysfi-chemfml.1.0.1
- ~~Add to snapshot `snapshot-stable-0-0-7`~~ (Inconsistent)

### CODE REVIEW

File: packages/satysfi-chemfml-doc/satysfi-chemfml-doc.1.0.0/opam [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+This package is a document of chemfml
+"""
+maintainer: "Amadeus_vn <git@amas.dev>"
+authors: "Amadeus_vn <git@amas.dev>"
+license: "MIT"
+homepage: "https://github.com/gw31415/satysfi-chemfml"
+dev-repo: "git+https://github.com/gw31415/satysfi-chemfml.git"
+bug-reports: "https://github.com/gw31415/satysfi-chemfml/issues"
+depends: [
+  "satysfi" { >= "0.0.6" & < "0.0.7" }
```
Comment (na4zagin3) [inline packages/satysfi-chemfml-doc/satysfi-chemfml-doc.1.0.0/opam:14]:
```suggestion
  "satysfi" { >= "0.0.6" & < "0.0.8" }
```

File: packages/satysfi-chemfml-doc/satysfi-chemfml-doc.1.0.1/opam [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+This package is a document of chemfml
+"""
+maintainer: "Amadeus_vn <git@amas.dev>"
+authors: "Amadeus_vn <git@amas.dev>"
+license: "MIT"
+homepage: "https://github.com/gw31415/satysfi-chemfml"
+dev-repo: "git+https://github.com/gw31415/satysfi-chemfml.git"
+bug-reports: "https://github.com/gw31415/satysfi-chemfml/issues"
+depends: [
+  "satysfi" { >= "0.0.6" & < "0.0.7" }
```
Comment (na4zagin3) [inline packages/satysfi-chemfml-doc/satysfi-chemfml-doc.1.0.1/opam:14]:
```suggestion
  "satysfi" { >= "0.0.6" & < "0.0.8" }
```

File: packages/satysfi-chemfml-doc/satysfi-chemfml-doc.1.0.1/opam [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  "satyrographos" { >= "0.0.2.6" & < "0.0.3" }
+
+  # You may want to include the corresponding library
+  "satysfi-chemfml" {= "%{version}%"}
+
+  # Other libraries
+  "satysfi-dist"
+  "satysfi-base"
+  "satysfi-easytable"
```
Comment (na4zagin3) [inline packages/satysfi-chemfml-doc/satysfi-chemfml-doc.1.0.1/opam:23]:
```suggestion
  "satysfi-easytable" {>= "1.1.2" & < "2"}
```

**Review by na4zagin3 — COMMENTED** [review]
Please support SATySFi 0.0.7 and declare lower version constraint on satysfi-easytable.

**Comment by na4zagin3** [comment]
Can you remove `satysfi-chemfml.1.0.0` from the MR?

**Comment by gw31415** [comment]
Thank you for advices!

I checked and fixed these issues.
Would you review commits?

Thanks.

**Comment by na4zagin3** [comment]
LGTM!  Thank you!

**Pull request merged**